src/Navigation/styled.js

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components"
+
+export const Wrapper = styled.div`
+    background-color: ${({ theme }) => theme.color.almostBlack};
+    min-height: 93px;
+    margin:0 auto;
+`;
+
+export const Container = styled.div`
+    max-width: 1300px;
```

Comment:
to me, in the design it looks like `max-width: 1368px;`

Reply:
thx for suggestion:) i fixed it!